packages/next/src/server/after/after-context.ts

```diff
@@ -123,26 +109,3 @@ function errorWaitUntilNotAvailable(): never {
     '`unstable_after()` will not work correctly, because `waitUntil` is not available in the current environment.'
   )
 }
-
-/** Disable mutations of `requestStore` within `after()` and disallow nested after calls.  */
-function wrapRequestStoreForAfterCallbacks(
-  requestStore: RequestStore
-): RequestStore {
-  return {
-    url: requestStore.url,
-    get headers() {
-      return requestStore.headers
-    },
-    get cookies() {
-      return requestStore.cookies
-    },
-    get draftMode() {
-      return requestStore.draftMode
-    },
-    // TODO(after): calling a `cookies.set()` in an after() that's in an action doesn't currently error.
-    mutableCookies: new ResponseCookies(new Headers()),
-    afterContext: requestStore.afterContext,
-    isHmrRefresh: requestStore.isHmrRefresh,
-    serverComponentsHmrCache: requestStore.serverComponentsHmrCache,
-  }
-}
```

Comment:
I'm probably missing something but I don't see where you track the transition to after via mutation and you no longer alter the request store bound to the callback

Reply:
Yea, this is just removing that part. It needs to be added back in a follow up.

Reply:
@gnoff  Yeah the plan is to have a mutable `phase` field on the store and set it when we enter `after`, i'll have a PR up soon